{"name": "test_add_with_type_error[1-1-2]", "status": "passed", "parameters": [{"name": "x", "value": "1"}, {"name": "y", "value": "'1'"}, {"name": "res", "value": "2"}], "start": 1718123091173, "stop": 1718123091173, "uuid": "c79760f4-1efa-4479-9c93-8868a0d6e132", "historyId": "0c76f3493f97f0f5c3ab99e7a750e14c", "testCaseId": "c4535174573fa79997bd065240c66b58", "fullName": "tests.tests_2024.test_main.TestCalculator#test_add_with_type_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('fixture_one', 'fixture_two')"}, {"name": "parentSuite", "value": "tests.tests_2024"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "TestCalculator"}, {"name": "host", "value": "ca8132c2c276"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_2024.test_main"}]}